write-fonts/src/tables/fvar.rs

```diff
@@ -8,6 +8,19 @@ pub use instance_record::InstanceRecord;
 include!("../../generated/generated_fvar.rs");
 
 impl Fvar {
+    /// We need everyone to have, or not have, post_script name so we can have a single record size.
+    pub fn check_instance_size(&self, ctx: &mut ValidationCtx) {
```

Comment:
@cmyr do these need to be public?

Reply:
That's a good point, let me see if anything kerplodes if not.

Reply:
It appears they do not, unless there's some reason someone would call them from The Outside. For now I will update to not be.